setup.py

```diff
@@ -32,6 +32,7 @@
     include_package_data=True,
     install_requires=[
         "astropy>=5,<6",
+        "attrs>=22.2.0",
         "click>=8,<9",
         "frictionless>=5.5.0,<6",
```

Comment:
This contains the same attrs hack that environment.yml has. Why don't we upgrade frictlionless here as well?

Reply:
see above